src/main/java/fr/lunatech/timekeeper/services/TimeSheetService.java

```diff
@@ -69,9 +69,12 @@ Long createDefaultTimeSheet(Project project, User owner, AuthenticationContext c
     @Transactional
     public Optional<Long> update(Long id, TimeSheetRequest request, AuthenticationContext ctx) {
```

Comment:
This method named as update().  May be I am missing something but what are we updating here and where is the the persist operation ?